bom/pom.xml

```diff
@@ -75,9 +76,12 @@
     <spring-security-crypto.version>5.2.4.RELEASE</spring-security-crypto.version>
     <truth.version>0.28</truth.version>
     <vertx.version>3.9.1</vertx.version>
-    <wildfly-common.version>1.5.1.Final</wildfly-common.version>
+    <wildfly-common.version>1.5.4.Final-format-001</wildfly-common.version>
```

Comment:
is this an *official* version?

Reply:
It is. I asked around when I first encountered it as it's purely documented. I contains just a couple of additional logging related classes, but it's needed for Quarkus.

BTW. I'm still to do IP due diligence for all this. That's one of the next tasks.